Source/Resources/Content/MSGraph/Groups/BUILTIN - Group Creators.json

```diff
@@ -1,6 +1,6 @@
 {
   "assignedLicenses": [],
-  "description": "Restrict Office 365 Group Creation to just users in this group.",
+  "description": "What is it: A manual group used to determine users that have the access to create Microsoft 365 groups. \n Simeon Notes: Simeon's baseline includes the user 'BUILTIN – Global Admin' as a member of this group.",
```

Comment:
**What does this do?**
Configures restrictions for creating Office 365 groups. The baseline restricts users not in this group from creating Office 365 groups. @metamenos is this Office 365 groups or Microsoft 365 groups?

**Why should you use this?**
If you want to tighten security around Office 365 group creation. 

**What is the end user impact?**
User who are not in this group will be restricted from creating Office 365 groups. 

**Learn more**
[Manage who can create Microsoft 365 Groups](https://docs.microsoft.com/en-us/microsoft-365/solutions/manage-creation-of-groups?view=o365-worldwide)

Reply:
This is only used for the Unified.groups policy below. Probably better to say Microsoft groups instead of Office. Could also repeat the link if we wanted: 
https://docs.microsoft.com/en-us/microsoft-365/solutions/manage-creation-of-groups?view=o365-worldwide

Reply:
@metamenos changed accordingly. do we need to change anything regarding this only being used for the 'Unified.groups' ?